tmb_follow/src/rto_tf_broadcaster.py

```diff
@@ -73,19 +73,17 @@ def handle_robot_pose(msg, robotname):
     br = tf.TransformBroadcaster()
     yaw = msg.yaw
     if robotname == "robot_blind":
-    quaternion = tf.transformations.quaternion_from_euler(0, 0, yaw)
-    br.sendTransform((msg.x, msg.y, 0),
-                     quaternion,
-                     rospy.Time.now(),
-                     robotname,
-                     "map")
+        quaternion = tf.transformations.quaternion_from_euler(0, 0, yaw)
```

Comment:
@parsaw100 okay so now if its the blind robot it will send, 
other wise it wont send anything .
can you please 

if robot blind:
    quaternion, send
 else:
     semd as normal